.github/workflows/main.yml

```diff
@@ -39,23 +39,42 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements-dev.txt
 
-      - name: Run pycodestyle
-        run: |
-          pycodestyle api/*.py
-
-      - name: Run pylint
-        run: |
-          pylint api.auth
-          pylint api.db
-          pylint -d R0903 api.models
-          pylint api.pubsub
-          pylint test
-          pylint api.main
-
```

Comment:
We need to keep this to report errors, the annotations only add annotations.

Reply:
That's what I mentioned in the earlier comment. I thought there should be a way to fail `lint` job.

Reply:
Anyway, I'll revert this.

Reply:
Yes I had a look at this when enabling it on kernelci-core and I didn't see an obvious way to make the annotation action fail the check if some linters reported any errors.  So for now we need to keep both actions.

Reply:
Oh, okay.